### ISSUE TITLE
2.1.0 release

### ISSUE THREAD

**Issue by adamralph** [adamralph/xbehave.net#266]
**Ready** when all other issues forming part of the release are **Done**.
- [x] ensure patches have been merged from master to dev
- [x] run code analysis in VS in _Release_ mode and address violations (send a regular PR which must be merged before continuing)
- [x] create and merge PR from upstream dev to master
- [x] smoke test NuGet package
- [x] create release in GitHub UI **on master** including releaseNotes, mentioning non-owner contributors, if any
- [x] push nuget package
- [x] copy release notes from GitHub to NuGet
- ~~de-list buggy nuget packages if present~~
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in https://gitter.im/xbehave/xbehave.net
- [x] post links to the NuGet release, GitHub release, and Tweet in each issue in this milestone, with thanks to contributors
- [x] increment minor version on branch from dev
- [x] create and merge PR to upstream dev
- [x] create a new milestone for the next release
- [x] create new issue based on this one for next release, adding it to the new milestone
- [x] update samples/raise issue to do so
- [x] close this milestone

**Comment by adamralph** [comment]
https://twitter.com/xbehavenet/status/725377198648098816
